src/main/scala/org/constellation/storage/TokenService.scala

```diff
@@ -0,0 +1,81 @@
+package org.constellation.storage
+
+import cats.effect.Concurrent
+import cats.effect.concurrent.Ref
+import cats.implicits._
+import io.chrisdavenport.fuuid.FUUID
+import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
+import io.circe.{Decoder, Encoder}
+import io.circe.generic.semiauto._
+import org.constellation.storage.TokenService.{Token, TokenVerificationResult}
+
+class TokenService[F[_]: Concurrent] {
```

Comment:
I would move it to `constellation.session` or sth instead of `constellation.storage` 🤔